provider/pkg/gen/schema.go

```diff
@@ -169,16 +172,16 @@ func (g *packageGenerator) findResources(parent string, resources map[string]dis
 		names = append(names, name)
 	}
 	sort.Strings(names)
-	for _, name := range names {
-		res := resources[name]
-		name = ToUpperCamel(inflector.Singularize(name))
+	for _, resourceName := range names {
+		res := resources[resourceName]
+		name := ToUpperCamel(inflector.Singularize(resourceName))
 		var createMethod, getMethod, updateMethod, deleteMethod *discovery.RestMethod
 		for methodName, value := range res.Methods {
 			restMethod := value
 			switch methodName {
 			case "create", "insert":
 				createMethod = &restMethod
-			case "update", "replaceService"/*special case for Cloud Run*/:
+			case "update", "replaceService" /*special case for Cloud Run*/ :
```

Comment:
It sounds like `replaceService` should become an override in your model

Reply:
Yeah if we go with the overrides, then since this affects just one service we can add it to overrides. I will wait for your thoughts on https://github.com/pulumi/pulumi-gcp-native/pull/33#discussion_r610073794 - perhaps the overrides is premature. It does make it a bit of a pain to comb through resources without known create methods.